src/server/game/Handlers/AuctionHouseHandler.cpp

```diff
@@ -208,6 +209,12 @@ void WorldSession::HandleAuctionSellItem(WorldPacket& recvData)
         if (itemEntry == 0)
             itemEntry = item->GetTemplate()->ItemId;
 
+        if (DisableMgr::IsDisabledFor(DISABLE_TYPE_ITEM, itemEntry, nullptr, ITEM_DISABLE_AUCTIONHOUSE))
```

Comment:
With this you completely change the old meaning of `AuctionHouseBot.forceExcludeItems` configuration, this check will not prevent regular player from ever listing that specific item on AH manually, you can't have this tied to AH bot configuration